From b4b402570ff6ee7b021eedb18d79287317147eb4 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwelk@xhochy.com>
Date: Mon, 19 Feb 2024 08:37:55 +0100
Subject: [PATCH] Only install asv sources

---
 pyproject.toml | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index a873536..407b09a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -131,6 +131,4 @@ archs = "x86_64 arm64"
 archs = ['AMD64']
 
 [tool.setuptools.packages.find]
-where = ["."]
-exclude = ["wheelhouse"]
-namespaces = true
+include = ["asv*"]
-- 
2.39.3 (Apple Git-145)

